src/components/Chip/Chip.scss

```diff
@@ -21,6 +21,38 @@ $vertical-padding: ($min-height - line-height(body) - rem(2px)) / 2;
     white-space: nowrap;
     vertical-align: middle;
     max-width: $maxWidth;
+
+    .chipButtonContent {
+        color: rgba(0,0,0,.54);
+        fill: rgba(0,0,0,.54);
+        padding: 0px;
+        min-width: auto;
+        min-height: auto;
+        margin:0px;
+        vertical-align: middle;
+        padding-top: 0.6rem;
+        padding-bottom: 0.5rem;
+        padding-left: 3.5rem;
+        position: relative;
+        z-index: 10;
```

Comment:
@Farhan-emgage is this z-index needed? I'm concerned it may lead to the chip displaying on top of other items such as dropdowns.

Reply:
@weegrom actually i placed this z index in respective to image  beside text. let me check is it creating issue with other component.

Reply:
@weegrom 

@emgage-smit has done the required changes